### ISSUE TITLE
Data Analyses / SCR / Generate Inputs - not able to generate input files

### ISSUE THREAD

**Issue by Shannon-Dubay** [pantheracorp/PantheraIDS_Issues#217]
Marike is posting on the forum about an urgent matter that needs urgent assistance (her words)
She is working on the latest IDS version with local databases and is not able to generate SCR input files. She says she has waited 2 hours and nothing has been generated
See post for log files: https://pantheraids.org/forum.html?p=post%2Fscr-input-files-not-generated-using-v-4-32-1026-11355460%3Fpid%3D1322214666

**Comment by Shannon-Dubay** [comment]
Update: Lauren is also experiencing this issue with server-based databases (S30_2020)

**Comment by PhilipFaure** [comment]
Hey @Shannon-Dubay is this still an issue? I can't recreate it on my machine, everything works as intended this side.

**Comment by Shannon-Dubay** [comment]
Yes, heres the thread on the forum: https://pantheraids.org/forum.html?p=post%2Fscr-input-files-not-generated-using-v-4-32-1026-11355460%3Fpid%3D1322235001
